spec/Subject-spec.ts

```diff
@@ -430,6 +430,31 @@ describe('Subject', () => {
     done();
   });
 
+  it('should expose observed status', (done) => {
+    const subject = new Subject();
+
+    expect(subject.isObeserved()).to.equal(false);
+
+    const sub1 = subject.subscribe(function (x) {
+      //noop
+    });
+
+    expect(subject.isObeserved()).to.equal(true);
+
+    const sub2 = subject.subscribe(function (x) {
+      //noop
+    });
+
+    expect(subject.isObeserved()).to.equal(true);
+    sub1.unsubscribe();
+    expect(subject.isObeserved()).to.equal(true);
+    sub2.unsubscribe();
+    expect(subject.isObeserved()).to.equal(false);
+    subject.unsubscribe();
+    expect(subject.isObeserved()).to.equal(false);
+    done();
```

Comment:
nit: I think the `done()` is not needed since this is a fully synchronous test?

Reply:
Fair, the one above it that I copied has it but others don't.  I've removed it.